stix_shifter_modules/alertflex/configuration/alertflex_config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "connection": {
+        "type": {
+            "default": "alertflex"
+        },
+        "host": {
+            "type": "text"
+        },
+        "port": {
+            "default": 8181,
+            "type": "number"
+        },
+        "help": {
+            "default": "<Help URL>",
```

Comment:
You may wish to include a link to any AlertFlex documentation around setting up a connection.

Reply:
Descriptions in configs are fixed, thank you.